compiler/rustc_hir_typeck/src/method/suggest.rs

```diff
@@ -3686,6 +3686,9 @@ impl<'a, 'tcx> FnCtxt<'a, 'tcx> {
             // same crate.
 
             let rcvr_ty = self.node_ty_opt(ty.hir_id);
+            let Some(rcvr_ty) = rcvr_ty else {
+                return;
```

Comment:
this return changes behavior instead of only returning from the fn itself? is this intentional?

Reply:
forget my question, I got it

Reply:
Solved